src/content/en/fundamentals/engage-and-retain/push-notifications/permissions-subscriptions.markdown

```diff
@@ -102,6 +101,10 @@ This is the result in Chrome.
 
 ![Chrome prompts for permissions.](images/news-permissions.png){:width="296px"}
 
+### What about the applicationServerKey? {#applicationserverkey}
+
+You probably noticed that the call to `subscribe()` contains a parameter called `applicationServerKey`. Its value is generated by your server. We were saving all server side issue for the next section. For now, there's one thing you need to know about the `applicationServerKey`: your app should pass key to the server as an array of eight-bit unsigned integers.
```

Comment:
The last paragraph could do with some tweaking.

"For now, there's one thing you need to know about the applicationServerKey: When passing in the key to a subscribe() call, make sure it's a Uint8Array (an array of eight-bit-unsigned integers)."

Main change - "should pass key to the server" <- This is incorrect. The browser passes it to the push service.

Secondary change is to simplify the eight-bit unsigned integers, it's not a specific as it could be.

Probably worth linking Uint8Array to: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array